### PR TITLE
Update/interval tweaks

### PR DESCRIPTION
**Pull request by sjauld**
AEMO::Market::Interval rewritten with new #parse_csv and #parse_json methods.

Added AEMO::NMI#current_annual_load

Minor bugfixes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# AEMO Gem Changelog
+
+## 1.2.0
```
Comment (jufemaiz) [inline CHANGELOG.md]: 0.2.0?

**Comment by jufemaiz** [comment]
Why did you remove coveralls?

**Comment by sjauld** [comment]
It's not in use? Doesn't simplecov replace it?

**Comment by jufemaiz** [comment]
Nope. It's used for collaboration – pushes go to Coveralls as part of CI.

https://coveralls.io/github/jufemaiz/aemo

Simplecov is for localised when you don't want to use a managed service
(which you do for Open Source).

:)

–
Joel Courtney
 m: +61 401 501 625
 e: euphemize@gmail.com
 w: http://euphemize.net
 p: http://flic.kr/jufemaiz/
 t: http://twitter.com/jufemaiz

File: Gemfile.lock [hunk 2]
```diff
@@ -75,22 +68,25 @@ GEM
       nokogiri (>= 1.5.10)
       rake
       rdoc
+      semver
     json (1.8.3)
     jwt (1.5.1)
-    listen (3.0.6)
```
Comment (jufemaiz) [inline Gemfile.lock]: Installing listen 3.1.5
Gem::InstallError: listen requires Ruby version >= 2.2.3, ~> 2.2.

https://travis-ci.org/jufemaiz/aemo/jobs/136015038

**Comment by jufemaiz** [comment]
https://github.com/jufemaiz/aemo/pull/19

**Pull request closed without merging**